TradeItIosTicketSDK2/TradeItYahooCryptoTradingTicketViewController.swift

```diff
@@ -492,13 +496,10 @@ class TradeItYahooCryptoTradingTicketViewController:
             guard let marketCell = cell as? TradeItSubtitleWithDetailsCellTableViewCell else { return cell }
             let quotePresenter = TradeItQuotePresenter(self.order.linkedBrokerAccount?.accountBaseCurrency)
             marketCell.configure(
```

Comment:
Since we don't have all the data from the crypto market service I left some labels blank. Let me know if you think of a value to add there.  
<img width="379" alt="screenshot 2018-06-21 10 02 32" src="https://user-images.githubusercontent.com/702474/41723940-63c7b6a2-753a-11e8-8299-b2b643c56995.png">

Reply:
can you add a note in the story to product about it